src/textures/Texture.js

```diff
@@ -299,6 +300,12 @@ Texture.prototype = Object.assign( Object.create( EventDispatcher.prototype ), {
 
 		return uv;
 
+	},
+
+	setInternalFormat: function(internalFormat) {
```

Comment:
I do not think we need this method. I disagree with @mrdoob. The exception is if it is needed for chaining in a coding pattern we recommend.

I think this method should be removed -- and discussed in a separate PR if you want.

Such methods should return `this` for chaining support, anyway.

Reply:
Yes forgot the `this`.
As it is an "advanced" usage, just changing the attribute manually is also fine I would say.